meso-org/meso-ui
# Discover view

## Description 
Created view and mock functionality of discover view. Check figma or slack to know which im talking about

## Test

* localhost:8080/#/discover
* Check viewpoints are ok
* press the filter button to have some mock filter options
* press save and they will show like in the mockup
